### ISSUE TITLE
Bold font for spanning_header

### ISSUE THREAD

**Issue by crow16384** [dbosak01/sassy#31]
Hello!

I would like to have bold face font for spanned header too, but can't figure out the way. Suggest a bug.

I did:
tbl <- create_table(out_example, first_row_blank = TRUE,
                    borders = c("top", "bottom"), header_bold = TRUE) |>
  spanning_header(from = `Placebo`, to=`Drug 160 mg`, "Day XXX")....

Header is not in bold, but all other header strings are.

**Comment by dbosak01** [comment]
Yes, you are right.  Looks like I forgot to add this feature.  Will get to it soon.

**Comment by crow16384** [comment]
Thank!

**Comment by crow16384** [comment]
Thanks!
